### PR TITLE
Website Updates

### PR DESCRIPTION
**Pull request by eteridvalishvili**
CDM Website Update - July 

- [x] Fix DPBE link
- [x] Update Overview Deck
- [x] Update the overview page with the two ISDA website pointer links and a pointer to the CDM interest form
- [x] Update Governance.md and Release.md with Brians changes
- [x] Create a materials page
- [x] Move cdm resources over from the FINOS landing page- 
- [x] Add a pointer for the materials page on the banner

Materials page: 

- [x] https://www.finextra.com/pressarticle/100503/six-major-banks-trial-fragmos-chain-in-otc-derivatives-market
- [x] https://www.securitiesfinancetimes.com/sltimes/SFT_issue_324.pdf
- [x] Landing Page Publications
- [x] Updated resources

Pointers to these links:

- [x] [https://www.isda.org/isda-digital-regulatory-reporting/](https://www.isda.org/2023/04/04/drr-infohub/)
- [x] https://www.isda.org/2023/02/16/isda-collateral-initiatives/
- [x] https://www.finos.org/common-domain-model
- [x] https://resources.finos.org/category/projects-sigs/common-domain-model-cdm?projects-sigs=common-domain-model-cdm&c=cG9zdDo5OTEwNzc%3D

### CODE REVIEW

File: docs/CDM-Collateral-WG.md
Comment (Oblongs) [inline docs/CDM-Collateral-WG.md:1]: The name of the guide is "Get Started with the CDM for Collateral" and should be used consistently.

File: docs/cdm-materials.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+### [ISLA, ICMA & ISDA Appoint FINOS for CDM Repository](https://www.islaemea.org/press-releases/isla-icma-isda-appoint-finos-for-cdm-repository/)
+
+On Thursday, September 8th, 2022, three leading organizations in the financial industry, International Securities Lending Association (ISLA), International Capital Market Association (ICMA), and International Swaps and Derivatives Association (ISDA), made a major announcement appointing FINOS to provide a repository for the Common Domain Model (CDM) with the aim of fostering growth in the open-source community.
+
+### [2022 Open Source in Finance Forum New York Keynote Panel: Common Domain Model (CDM)](https://resources.finos.org/content/keynote-panel-common-domain-model-cdm/?projects-sigs=common-domain-model-cdm)
+
+At the keynote session of the 2022 Open Source in Finance Forum New York, a panel of industry leaders including Adrian Dale of ISLA, Allan Milligan of ISDA, and Georgina Jarrett, joined moderator Ffion Acclan of Goldman Sachs to delve into the exciting topic of trade association collaboration, open sourcing, and the creation of a common domain for the market, built by the market.
```
Comment (chrisisla) [inline docs/cdm-materials.md:48]: "Alan" not "Allan"

File: docs/cdm-materials.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+The second half of the one-hour CDM presentation at the Open Source in Finance Forum (OSFF) in New York was dedicated to exploring the potential of Common Domain Model (CDM) in market infrastructure. A panel of experts, including Jon Nance of Axoni, Dianne Boyle of Symbiont, Sunil Challa of Credit Suisse, and Eleanor Acuna of ISDA, came together to delve into the possibilities of CDM in the real world. They discussed the practical applications and potential of CDM, offering valuable insights into the future of this exciting technology.
+
+### [FINOS on what’s next for CDM and open source trends](https://finadium.com/sfm-interview-finos-on-whats-next-for-cdm-and-open-source-trends/)
+
+On January 11th, 2023, FINADIUM sat down with Gabriel Columbro, the Executive Director at FINOS and General Manager of the Linux Foundation Europe, to discuss the exciting world of open source trends in financial services. During the conversation, Gabriel shared his insights on what to expect in the coming months and years and provided a broad overview of the overarching open source trends in the financial services industry.
```
Comment (chrisisla) [inline docs/cdm-materials.md:60]: "Gabriele" not "Gabriel"

File: website/versioned_docs/version-5.13.0/cdm-materials.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+### [ISLA, ICMA & ISDA Appoint FINOS for CDM Repository](https://www.islaemea.org/press-releases/isla-icma-isda-appoint-finos-for-cdm-repository/)
+
+On Thursday, September 8th, 2022, three leading organizations in the financial industry, International Securities Lending Association (ISLA), International Capital Market Association (ICMA), and International Swaps and Derivatives Association (ISDA), made a major announcement appointing FINOS to provide a repository for the Common Domain Model (CDM) with the aim of fostering growth in the open-source community.
+
+### [2022 Open Source in Finance Forum New York Keynote Panel: Common Domain Model (CDM)](https://resources.finos.org/content/keynote-panel-common-domain-model-cdm/?projects-sigs=common-domain-model-cdm)
+
+At the keynote session of the 2022 Open Source in Finance Forum New York, a panel of industry leaders including Adrian Dale of ISLA, Allan Milligan of ISDA, and Georgina Jarrett, joined moderator Ffion Acclan of Goldman Sachs to delve into the exciting topic of trade association collaboration, open sourcing, and the creation of a common domain for the market, built by the market.
```
Comment (chrisisla) [inline website/versioned_docs/version-5.13.0/cdm-materials.md:48]: Alan again
Actually it's Ffion Acland - same will be wrong on the previous file too

File: website/versioned_docs/version-5.13.0/cdm-materials.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+### [Implementing CDM: How is it Done, Why is it Beneficial to Financial Markets Infrastructure?](https://resources.finos.org/content/implementing-cdm-how-is-it-done-why-is-it-beneficial-to-financial-markets-infrastructure/?projects-sigs=common-domain-model-cdm)
+
+The second half of the one-hour CDM presentation at the Open Source in Finance Forum (OSFF) in New York was dedicated to exploring the potential of Common Domain Model (CDM) in market infrastructure. A panel of experts, including Jon Nance of Axoni, Dianne Boyle of Symbiont, Sunil Challa of Credit Suisse, and Eleanor Acuna of ISDA, came together to delve into the possibilities of CDM in the real world. They discussed the practical applications and potential of CDM, offering valuable insights into the future of this exciting technology.
```
Comment (chrisisla) [inline website/versioned_docs/version-5.13.0/cdm-materials.md:56]: Eleonora Acuna

File: website/versioned_docs/version-5.13.0/cdm-materials.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+### [FINOS on what’s next for CDM and open source trends](https://finadium.com/sfm-interview-finos-on-whats-next-for-cdm-and-open-source-trends/)
+
+On January 11th, 2023, FINADIUM sat down with Gabriel Columbro, the Executive Director at FINOS and General Manager of the Linux Foundation Europe, to discuss the exciting world of open source trends in financial services. During the conversation, Gabriel shared his insights on what to expect in the coming months and years and provided a broad overview of the overarching open source trends in the financial services industry.
```
Comment (chrisisla) [inline website/versioned_docs/version-5.13.0/cdm-materials.md:60]: Gabriel

**Review by chrisisla — CHANGES_REQUESTED** [review]
Can you check all the names are spelt correctly please? I have highlighted a few but there could be more.

File: docs/cdm-materials.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+### [CDM Now Available In Python](https://www.finos.org/blog/cdm-now-available-in-python)
+
+A recent contribution to the CDM project makes it available in the Python programming language for the first time, which will allow a new community of developers to use the CDM in their projects.
+
+### [Common Domain Model into Open Source at FINOS](https://www.finos.org/hubfs/Accepted%20FINOS%20Edits_%20Common%20Domain%20Model%20into%20Open%20Source%20at%20FINOS.docx.pdf)
+
+Read more about CDM's journey into open source, project contribution guidelines, principles, and frequently asked questions.
+
+### [Barclay's demonstration of interest rate swap processing using the Common Domain Model (CDM)](https://www.youtube.com/watch?v=LTWn_AWC4F8)
```
Comment (Oblongs) [inline docs/cdm-materials.md:38]: No quote in Barclays

**Review by chrisisla — APPROVED** [review]

**Review by Oblongs — APPROVED** [review]

**Pull request merged**